### PR TITLE
Sdk refactoring simple helpers

### PR DESCRIPTION
**Pull request by Velenir**
- Shuffled code around, so that every little file isn't directly in `src/*` and to make future functionality extensions easier
- Added `constructSimpleSDK`, which enables can provide API methods with minimal config.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,98 @@ yarn add @paraswap/sdk
 
 ## Using ParaSwap SDK
 
-### Import the necessary functions
+There are multiple ways to use ParaSwap SDK, ranging from simple construct-and-use approach to a fully composable _bring what you need_ way that allows for advanced tree-shaking a minimizes bundle size.
+
+### Simple SDK
+
+Can be created by providing `network` and either `axios` or `window.fetch` (or alternative `fetch` implemetation). Resulting SDK will be able to use all methoda that query the API.
```
Comment (alexshchur) [inline README.md:30]:
```suggestion
Can be created by providing `network` and either `axios` or `window.fetch` (or alternative `fetch` implementation). Resulting SDK will be able to use all methods that query the API.
```

File: README.md [hunk 2]
```diff
@@ -23,7 +23,98 @@ yarn add @paraswap/sdk
...
+    const DAI_CONTRACT = new ethers.Contract(DAI, ERC20_ABI, ethersSignerOrProvider)
+
+    const tx = await DAI_CONTRACT.approve(TransferProxy, amountInWei)
+
+    const txReceipt = await tx.wait(1)
+  }
+
+```
+
+If additionally provided `providerOptions` as the second parameter, the resulting SDK will also be able to approve Tokens for swap.
```
Comment (samthomson) [inline README.md:89]:
```suggestion
If optional `providerOptions` is provided as the second parameter, then the resulting SDK will also be able to approve Tokens for swap.
```

File: README.md [hunk 3]
```diff
@@ -23,7 +23,98 @@ yarn add @paraswap/sdk
...
+  // or
+  const paraSwapMin = constructSimpleSDK({network: 1, fetch: window.fetch})
+
+  async function swapExample() {
+    const signer: JsonRpcSigner = ...
+    const senderAddress = signer.address
+
+    const priceRoute = await paraSwapMin.getRate({
+      srcToken: ETH,
+      destToken: DAI,
```
Comment (samthomson) [inline README.md:47]: perhaps [define](https://github.com/paraswap/paraswap-sdk/blob/sdk-refactoring_simple_helpers/tests/simple.test.ts#L28-L29) these above or stringify (eg `'ETH'` `'DAI'`) to leave no uncertainty for the user/dev.

**Review by samthomson — APPROVED** [review]
Really nice work. I tested out some of it, but not all. When it comes closer to release/master I can test more exhaustively.

File: README.md (hunk 3 above)
Comment (Velenir) [reply README.md:50]: Done [ab6bf97](https://github.com/paraswap/paraswap-sdk/pull/123/commits/ab6bf976a5130b611ea55eaca64bb08f46fbef55)

**Pull request merged**